language-design.md

```diff
@@ -59,50 +72,45 @@ What does the post on grayscale tell us about the process of API design?
 
 **Response**
 
+It certainly highlights the process of identifying tradeoffs when attempting to optimize for multiple design goals.  This is noticeable in the tension between familiarity/precision (using `RGB(x,x,x)` for a particular level of grey) and more specific but less precise name (`gray(x)` (does a higher value mean lighter or darker?)). 
 
+The comments are also quite interesting, as we see that one individual suggests using the common `greyscale()` term to name the function. Although it is not inherently any clearer than `gray()`, people are used to seeing and using it. However, as we can see from a later commenter,the function `greyscale()` is already defined by [Filter Effects](https://drafts.fxtf.org/filters/#grayscaleEquivalent). This namespace conflict might suggest suboptimal design as well.
```

Comment:
Again another kind of random question that this made me think of:

The proposed function is `greyscale()` which could be a good term since it pattern matches with other languages (pattern matching across languages was not talked about too much but I think it is very important to language design). What I wonder is if there were to be a `greyscale()` function then should there also be a `grayscale()` function (notice its grAy not grEy) ? This question extends to, should there be various function names depending on what vocal language someone speaks? 

Ross and I wondered about if popular programming languages came in different language versions and my initial research seems to show that comments might change languages but if statements are still "if" not "si" ... which is interessting?.